opencontext_py/tests/unit/searcher/new_solrsearcher/test_sorting.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import logging
+from opencontext_py.apps.searcher.new_solrsearcher import configs
+from opencontext_py.apps.searcher.new_solrsearcher.sorting import SortingOptions
+
+logger = logging.getLogger("tests-unit-logger")
+
+
+TESTS_REQUEST_TO_SOLR = [
```

Comment:
In this set of test cases, the following sortable fields show up:

scores to sort by:

* `interest_score`
* `sort_score`
* `slug_type_uri_label`
* `published`

This list isn't meant to be exhaustive, right -- since the focus of this unit test must be on the syntax of specifying sort key[s] in various ways and in singular or multiple formats.  That is, it's a test of the logic at https://github.com/ekansa/open-context-py/pull/752/files/a1e3ff267ef86abcf5fb78aa193aba07eaba59e4#diff-48fa456f707079f42e5477246111fe6eR64 ?

Reply:
We don't have many sorting options, because Solr only allows (or at least it used to allow) sorting on fields that get a single value. Most solr fields in Open Context allow multiple values, so are not sortable.